### DISCUSSION TITLE
Dockerfile for both Radarr & Sonarr for Recyclarr Dotnet Error

### DISCUSSION THREAD

**Discussion by baxterblk** [RandomNinjaAtk/arr-scripts#296] [Q&A]
### [Dockerfile]

```
FROM lscr.io/linuxserver/radarr:latest # change to sonarr if using sonarr


# Install dependencies and .NET runtime
RUN apk add --no-cache \
    icu-libs \
    krb5-libs \
    libgcc \
    libintl \
    libssl3 \
    libstdc++ \
    zlib \
    && mkdir -p /run/radarr-temp \
    && wget https://dot.net/v1/dotnet-install.sh -O dotnet-install.sh \
    && chmod +x dotnet-install.sh \
    && ./dotnet-install.sh --channel 8.0 --install-dir /usr/share/dotnet \
    && rm dotnet-install.sh

# Ensure the dotnet path is added
ENV PATH="/usr/share/dotnet:$PATH"
```

**Comment by RandomNinjaAtk** [comment]
Not sure how this is relevant, donet is installed as part of the setup script... So  what your trying to do is probably not needed...

see here:
https://github.com/RandomNinjaAtk/arr-scripts/blob/8d9271f31d8282a3c7bbfc1eea008d631b5ce70e/sonarr/setup.bash#L43

**Reply by baxterblk** [reply]
Recyclarr would not install no matter how many times I tried.


I've used these scripts across numerous machines and never had an issue. 

When the issue came up, I simply made the Dockerfiles.

**Reply by RandomNinjaAtk** [reply]
I believe the recyclarr issues was something that was discovered recently and resolved.

**Reply by baxterblk** [reply]
Sadly, it has not been resolved.

**Reply by RandomNinjaAtk** [reply]
Please provide logs showing the problem then, because it’s working fine in my testing and isn't reproducible..
